### PR TITLE
PR - str test case, shortcut

### PR DESCRIPTION
**Pull request by joyact**

### CODE REVIEW

File: src/runtime/runtime.ts [hunk 1]
```diff
@@ -48,19 +48,19 @@ function matchMediaPreference() {
 // prettier-ignore
 function isKeyDownDarkMode(e: KeyboardEvent) {
 	const ok = (
-		e.ctrlKey &&
+		!e.ctrlKey &&
 		!e.altKey &&
-		(e.key.toLowerCase() === "d" || e.keyCode === 68)
+		(e.key.toLowerCase() === "`" || e.keyCode === 68)
```
Comment (zaydek) [inline src/runtime/runtime.ts:53]: Hey this is pretty close but we need to change `68` to `192`. You can use keycode.info as a reference.

File: tests/utils/str-ends-with.test.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+declare function sass(data: string): string
+declare function errSass(testStr: string): Function
+
+test("integration", () => {
+	expect(errSass(`str-ends-with("Hello, world!", "?")`)).toThrowError("false")
+	expect(errSass(`str-ends-with("Hello, world!", "!")`)).toThrowError("true")
+	expect(errSass(`str-ends-with("", "")`)).toThrowError("true")
+	expect(errSass(`str-ends-with(123, 3)`)).toThrowError("true")
+	expect(errSass(`str-ends-with("Typescript", "script")`)).toThrowError("true")
+})
```
Comment (zaydek) [inline tests/utils/str-ends-with.test.ts:13]: Make sure you add `Files: Insert Final Newline` in VSCode settings (for the EOF).

File: tests/utils/str-replace.test.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/**
+ * @jest-environment node
+ */
+declare function sass(data: string): string
+declare function errSass(testStr: string): Function
+
+test("integration", () => {
+	expect(errSass(`str-replace("Hello, world!", "Hello", "Bye")`)).toThrowError("Bye, world!")
```
Comment (zaydek) [inline tests/utils/str-replace.test.ts:8]: In the future try to add a test for `""` or `null` for example.

File: tests/utils/str-replace.test.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+	expect(errSass(`str-replace("Javascript", "Java", "Type")`)).toThrowError("Typescript")
+})
```
Comment (zaydek) [inline tests/utils/str-replace.test.ts:10]: EOF! 😡

File: tests/utils/str-starts-with.test.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+declare function sass(data: string): string
+declare function errSass(testStr: string): Function
+
+test("integration", () => {
+	expect(errSass(`str-starts-with("Hello, world!", "?")`)).toThrowError("false")
+	expect(errSass(`str-starts-with("Hello, world!", "H")`)).toThrowError("true")
+	expect(errSass(`str-starts-with("", "")`)).toThrowError("true")
+	expect(errSass(`str-starts-with(123, 1)`)).toThrowError("true")
+	expect(errSass(`str-starts-with("Typescript", "Type")`)).toThrowError("true")
+})
```
Comment (zaydek) [inline tests/utils/str-starts-with.test.ts:13]: EOF

File: tests/utils/str-replace.test.ts (hunk 3 above)
Comment (zaydek) [inline tests/utils/str-replace.test.ts:8]: In the future try to add a test for `""` or `null` for example.

File: tests/utils/str-replace.test.ts (hunk 4 above)
Comment (zaydek) [inline tests/utils/str-replace.test.ts:10]: EOF! 😡

File: tests/utils/str-ends-with.test.ts (hunk 2 above)
Comment (zaydek) [inline tests/utils/str-ends-with.test.ts:13]: Make sure you add `Files: Insert Final Newline` in VSCode settings (for the EOF).

File: src/runtime/runtime.ts (hunk 1 above)
Comment (zaydek) [inline src/runtime/runtime.ts:53]: Hey this is pretty close but we need to change `68` to `192`. You can use keycode.info as a reference.

**Review by zaydek — CHANGES_REQUESTED** [review]
